msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -1189,7 +1191,7 @@ public static List<Map.Entry<String, String>> appendToExtraQueryParametersIfWebA
             @Nullable final List<Map.Entry<String, String>> queryStringParameters,
             @NonNull final PublicClientApplicationConfiguration configuration) {
         //Putting behind passkey feature flag until feature is ready.
-        if (FidoConstants.IS_PASSKEY_SUPPORT_READY
+        if (CommonFlightManager.isFlightEnabled(CommonFlight.ENABLE_PASSKEY_FEATURE)
```

Comment:
I don't think MSAL side can be rolled back remotely.

Reply:
True, and the feature is now slated for next release, so I'm going to remove this flag from MSAL in general.